src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -397,7 +397,7 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
 
   private async _copyYaml() {
     if (this._editor?.yaml) {
-      copyToClipboard(this._editor.yaml);
+      await copyToClipboard(this._editor.yaml);
```

Comment:
If we don't do anything after, there is no need to await it, we should just await it when we are going to show a toast that it is done etc.